tests/test_observing_block.py

```diff
@@ -94,6 +94,14 @@ def test_seeing(self):
             SeeingConstraint(max=-0.1)
 
 
+class TestObservingScript(unittest.TestCase):
+    def test_get_script_configuration(self):
+        script = ObservingScript(name="slew", standard=True, parameters={"target": "W48"})
+
+        script_config = script.get_script_configuration()
+        assert script_config == "target: W48\n"
```

Comment:
I wrote all the tests with unittest and not pytest so this looks a bit odd, but obviously you can change everything here to the T&S testing standard.